doc/user/install-operator-sdk.md

```diff
@@ -0,0 +1,75 @@
+# Install the Operator SDK CLI
+
+## Install from release
+
+### Linux
+
+```sh
+curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.7.0/operator-sdk-v0.7.0-x86_64-linux-gnu && chmod +x operator-sdk && sudo cp operator-sdk /usr/local/bin/ && rm operator-sdk
+```
+
+### OSX
+
+```sh
+curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.7.0/operator-sdk-v0.7.0-x86_64-apple-darwin && chmod +x operator-sdk && sudo cp operator-sdk /usr/local/bin/ && rm operator-sdk
+```
+
+#### Verifying the downloaded binary
+
+```sh
+# OSX
+curl -OJ https://github.com/operator-framework/operator-sdk/releases/download/v0.7.0/operator-sdk-v0.7.0-x86_64-apple-darwin.asc
+# Linux
+curl -OJ https://github.com/operator-framework/operator-sdk/releases/download/v0.7.0/operator-sdk-v0.7.0-x86_64-linux-gnu.asc
+```
+
+To verify a release binary using the provided asc files, place the binary and corresponding asc
+file into the same directory and use the corresponding command:
+
+```sh
+# macOS
+$ gpg --verify operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin.asc
+# GNU/Linux
+$ gpg --verify operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu.asc
+```
+
+If you do not have the maintainers public key on your machine, you will get an error message similar
+to this:
+
+```sh
+$ git verify-tag ${TAG_NAME}
```

Comment:
The `$ git verify-tag ${TAG_NAME}` line doesn't make sense in this context, so I think we can just remove it, leaving the lines prefiexed with `gpg:` (which still DO make sense)

However, the `git verify-tag` failure this describes does still make sense in the release doc. Maybe we should duplicate this one bit about the failed gpg check in both docs?